src/app/lib/analyticsUtils/index.js

```diff
@@ -333,4 +333,272 @@ export const getCampaignType = () => {
   return null;
 };
 
+const getMarketingUrlParam = (query, field) =>
+  Object.prototype.hasOwnProperty.call(query, field) ? query[field] : '';
+
+const buildMarketingString = marketingValues =>
+  marketingValues
+    .map(({ value, wrap }) => (wrap && value ? `[${value}]` : value))
```

Comment:
Just a small question about wrapping some `medium` parameter values in square brackets. As I can see in the code some of the values in the key:value pairs will be wrapped, but the campaign url example has no square brackets:
```
/persian?at_medium=affiliate&at_campaign=73&at_type=partner&at_identifier=whatsapp&at_format=Link&at_creation=wspersian&at_variant=Editorial
```

Are we 100% sure they need wrapping in square brackets?

Reply:
@HarveyPeachey in the original issue, it was stated that they be wrapped in square brackets. kind of like some of the values in the `xtor` url format ie. `xtor=AD-3030-[ad_version7]-[without_text]-[468]-[www.site.com]-[GT]-[top_page]`

Reply:
In the case of `at_medium`, the strings arrive without square brackets, and we need to add them